### PR TITLE
Add applications

### PR DESCRIPTION
**Pull request by natebrennand**
Need to add a test for using it.

https://www.twilio.com/docs/api/rest/applications

relevant code in this: https://github.com/natebrennand/twiliogo/commit/f3b165aeb1f313e01eabf34729e14f6c9bc8206d

### CODE REVIEW

**Comment by natebrennand** [comment]
wait on an example program that uses it

**Comment by natebrennand** [comment]
@chelpu, good for review & merge if it lgty

File: applications/applications.go [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+	}
+
+	if m.FriendlyName != "" {
+		v.Add("FriendlyName", m.FriendlyName)
+	}
+	if m.APIVersion != "" {
+		v.Add("APIVersion", m.APIVersion)
+	}
+	if m.VoiceURL != "" {
+		v.Add("VoiceURL", m.VoiceURL)
```
Comment (chelpu) [inline applications/applications.go:113]: Do caps on URL matter here?

File: applications/applications.go [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+	v := url.Values{}
+	if r.FriendlyName != "" {
+		v.Add("FriendlyName", r.FriendlyName)
+	}
+
+	if r.FriendlyName != "" {
+		v.Add("FriendlyName", r.FriendlyName)
+	}
+	if r.APIVersion != "" {
+		v.Add("APIVersion", r.APIVersion)
```
Comment (chelpu) [inline applications/applications.go:208]: Or here?

**Comment by chelpu** [comment]
If the answers to the caps thing is "no", it's ready to merge.

File: applications/applications.go (hunk 1 above)
Comment (natebrennand) [inline applications/applications.go:113]: Oops, yes, same for URI. Probably have this mistake in other URL encoders
too... will fix once I get in today

**Pull request merged**